File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    s = s.toLowerCase();
    let countP = 0;
    let countY = 0;
        
    for (const c of s) {
        if (c == 'p') countP++;
        if (c == 'y') countY++;
    }

    return countP == countY ? true : false;
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <algorithm>
#include <cctype>
using namespace std;

bool solution(string s)
{
    transform(s.begin(), s.end(), s.begin(), ::tolower);
    int countP = count(s.begin(), s.end(), 'p');
    int countY = count(s.begin(), s.end(), 'y');
    return countP == countY ? true : false;

}